tasmota/tasmota_xdrv_driver/xdrv_100_modbus_bridge.ino

```diff
@@ -0,0 +1,455 @@
+/*
+  xdrv_100_modbus_bridge.ino - modbus bridge support for Tasmota
+
+  Copyright (C) 2021  Theo Arends and Dániel Zoltán Tolnai
+
+  This program is free software: you can redistribute it and/or modify
+  it under the terms of the GNU General Public License as published by
+  the Free Software Foundation, either version 3 of the License, or
+  (at your option) any later version.
+
+  This program is distributed in the hope that it will be useful,
+  but WITHOUT ANY WARRANTY; without even the implied warranty of
+  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+  GNU General Public License for more details.
+
+  You should have received a copy of the GNU General Public License
+  along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+/*
+  Example Command: modbussend {"deviceaddress": 1, "functioncode": 3, "startaddress": 1, "type":"uint8", "count":4}
+*/
+
+#ifdef USE_MODBUS_BRIDGE
+/*********************************************************************************************\
+ * Modbus Bridge using Modbus library (TasmotaModbus)
+\*********************************************************************************************/
+
+#define XDRV_100 100
+#define HARDWARE_FALLBACK 2
+#define MBR_MAX_VALUE_LENGTH 30
+#define MBR_SPEED TM_MODBUS_BAUDRATE
+
+#define D_CMND_MODBUS_SEND "ModbusSend"
+#define D_CMND_MODBUS_SETBAUDRATE "ModbusSetBaudrate"
+#define D_CMND_MODBUS_SETSERIALCONFIG "ModbusSetSerialConfig"
+
+#define D_JSON_MODBUS_RECEIVED "ModbusReceived"
+#define D_JSON_MODBUS_DEVICE_ADDRESS "DeviceAddress"
+#define D_JSON_MODBUS_FUNCTION_CODE "FunctionCode"
+#define D_JSON_MODBUS_START_ADDRESS "StartAddress"
+#define D_JSON_MODBUS_COUNT "Count"
+#define D_JSON_MODBUS_ENDIAN "Endian"
+#define D_JSON_MODBUS_TYPE "Type" // allready defined
+#define D_JSON_MODBUS_VALUES "Values"
+#define D_JSON_MODBUS_LENGTH "Length"
+
+const char kModbusBridgeCommands[] PROGMEM = "|" // No prefix
+    D_CMND_MODBUS_SEND "|" D_CMND_MODBUS_SETBAUDRATE "|" D_CMND_MODBUS_SETSERIALCONFIG;
```

Comment:
Use prefix ``Modbus`` and remove from command names

Reply:
I used https://github.com/arendst/Tasmota/blob/development/tasmota/tasmota_xdrv_driver/xdrv_08_serial_bridge.ino as an example, I don't understand what I have to do for this.

Reply:
Do you want me to rename them to:

D_MODBUS_LENGTH
D_MODBUS_SEND
D_MODBUS_SETBAUDRATE

?

Reply:
Ok thank you!

I will try to finish documentation this week.